swarm/nodeinfo.go

```diff
@@ -25,6 +27,26 @@ type NodeInfo struct {
 	Port uint16
 }
 
+func NewStaticNodeInfo(name, addr string) *NodeInfo {
+	a := strings.Split(addr, ":")
```

Comment:
this split won't work with IPv6 addresses. We should use https://godoc.org/net#SplitHostPort